backend/package-lock.json

```diff
@@ -18,7 +18,8 @@
         "node-worker-threads-pool": "^1.5.1",
         "socks-proxy-agent": "^6.2.0",
         "typescript": "~4.7.2",
-        "ws": "~8.7.0"
+        "ws": "~8.7.0",
+        "zeromq": "^6.0.0-beta.6"
```

Comment:
NACK for adding a new third party npm module dependency

Reply:
is there something wrong specifically with the zeromq package? like the others are also 3rd party packages